lib/chef/chef_fs/file_system/chef_server/cookbook_file.rb

```diff
@@ -69,7 +69,7 @@ def compare_to(other)
           private
 
           def calc_checksum(value)
-            OpenSSL::Digest.hexdigest("MD5", value)
+            OpenSSL::Digest::MD5.hexdigest(value)
```

Comment:
can we just call `::Digest::MD5.hexdigest` here instead?

Reply:
(similarly obviously for all the other MD5 calls?)

Reply:
@jaym any recollection of that choice?

Reply:
https://github.com/chef/chef/pull/4481/files#diff-d19a42ae266a2919d7cfeb43c77dbaffR917
I'm guessing its relying on a similar pattern to that. So doing what lamont said would preserve the old behavior

Reply:
Oh I bet that breaks in Ruby 3 as well, but at least being consistent would funnel everything through that one const_set